### DISCUSSION TITLE
Use Bing/Cortana (or simular platform) searching system to provide improved search

### DISCUSSION THREAD

**Discussion by jonathan-rogers-dev** [bonzibudd/Fluent-Metro#129] [Ideas]
Is it possible to use the integrated searching system in Windows 10 (Bing/Cortana) or a third-party indexing and searching system in Fluent Metro to improve searching from the start menu? @bonzibudd

**Comment by bonzibudd** [comment]
I don’t think that’s possible to implement right now, but I do think that could be useful. For now, the closest thing is that you can remove the search box in Open-Shell settings and put the Windows search bar on the Taskbar.
